client/src/components/Dashboard/Dashboard.js

```diff
@@ -4,26 +4,61 @@ import { toast } from 'react-toastify';
 import completeWorkflowToast from '../../utils/Toasts/default';
 import * as actions from '../../actions/index';
 import { connect } from 'react-redux';
+import './Dashboard.css';
 
 class Dashboard extends Component {
+    constructor(props) {
+        super(props);
+
+        // Set music state to pause at the beginning of session
+        this.state = { isPlaying: false };
+        this.togglePlay = this.togglePlay.bind(this);    
+    }
 
     componentDidMount() {
         this.props.showHeader();
         // Update the user model so that components like the Profile can pre-load the up-to-date model
         this.props.fetchUser();
-        
+
         var { location } = this.props;
         // Toast if the user just finished the workflow
         if (location.state && location.state.finishedWorkflow) {
             toast("You have completed the setup!");
         }
     }
 
+    // Toggle playing state
+    togglePlay() {
+        var currentState = this.state.isPlaying;
+        this.setState({ isPlaying: !currentState });
+    }
+
     render() {
         return (
-            <div className="container">
-                <h2>Dashboard</h2>
+            <div className="container-bg">      
                 {completeWorkflowToast}
+
+                <div className="carousel-suggested-container-container">
+                        <div className="carousel-suggested-container">
+                            <div className="background-gradient"> 
+                                <div className="song-player-container">
+                                    <div className="artwork-wrapper">
+                                        <img src={require('../../assets/album-temp.png')}/>
+                                    </div>
+                                </div>
+                                <div className="username-title-container"> 
+                                    <div className="track-title">Shipwreck - The Coast</div>
+                                    <div className="track-username">testuser201</div>
+                                </div>
+                                    <div className="song-player">
+                                        <div onClick={this.togglePlay} className={"button " + (this.state.isPlaying ? 'play': 'pause')}></div>
+                                    </div> 
+                                <div>
```

Comment:
I've cleaned up the indentation of the nested divs here to make them more readable.